### ISSUE TITLE
[BUG/ISSUE] (Enter your description here)

### ISSUE THREAD

**Issue by rumplin** [marticliment/WingetUI#757]
## Before submitting the issue

- [x ] **I have checked that the bug is not a duplicate or, if it is, it needs to be reopened**
- [x ] **I have checked that I am running the latest version of WingetUI**
- [x ] **I have tried resetting and reinstalling WingetUI and the issue persisted or happened again**
- [x ] **I have checked the [FAQ](https://github.com/martinet101/WingetUI#faq) section for solutions**
- [x ] **I have read the [contributing guidelines](https://github.com/marticliment/WingetUI/blob/main/CONTRIBUTING.md#issues) and I agree with the [Code of Conduct](https://github.com/marticliment/WingetUI/blob/main/CODE_OF_CONDUCT.md)**

## Describe the bug
WingetUI shows that I can update some Windows Runtimes.
But in the end it's not able to update.

## To Reproduce
Steps to reproduce the behaviour:
1. Have older Windows Runtimes installed
2. Run WingetUI
3. It will detect that it can update some Runtimes
4. It will fail to update the runtimes

## Expected behaviour
Either update the runtimes and uninstall old one or don't show that there are updates if I already have the latest runtime also installed

## Screenshots
Windows 11 machine

This is what is currently installed on the machine:
![ss 2023-02-28 083843](https://user-images.githubusercontent.com/35449956/221785813-51615618-b355-460e-98a1-d7639f680986.png)
This is what WingetUI shows:
![ss 2023-02-28 083436](https://user-images.githubusercontent.com/35449956/221785815-a635fd1a-1bfa-4397-8dce-a888c9fec82c.png)
This is where it fails:
![ss 2023-02-28 083512](https://user-images.githubusercontent.com/35449956/221785808-76e16e00-7499-440d-9bce-0c7ef97d2aa0.png)

## The Log
<details>
[winget.txt](https://github.com/marticliment/WingetUI/files/10847705/winget.txt)
</details>

**Comment by marticliment** [comment]
Does this work with winget command-line?

**Comment by rumplin** [comment]
I can try, just give me the command to run.

**Comment by rumplin** [comment]
```
C:\Users\waldo>winget upgrade
Name                                             Id                                Version Available Source
-----------------------------------------------------------------------------------------------------------
Microsoft Windows Desktop Runtime - 6.0.11 (x86) Microsoft.DotNet.DesktopRuntime.6 6.0.11  6.0.14    winget
1 upgrades available.

```

**Comment by marticliment** [comment]
On the updates section, right-click the failing packages -> package info, and copy the command shown below.
Run that command in a cmd window

**Comment by rumplin** [comment]
```
C:\Users\waldo>winget update --id Microsoft.DotNet.DesktopRuntime.6 --exact --source winget --accept-source-agreements --force
No installed package found matching input criteria.

```

**Comment by rumplin** [comment]
Seems that it's not a WingetUI issue: https://github.com/microsoft/winget-cli/issues/2345

**Comment by marticliment** [comment]
Yes, and it seems like they are going to fix that before winget 1.5.
